cache/test_purge.py

```diff
@@ -203,16 +207,54 @@ def set_cache_val(self, cache_val):
         self.start_all_services()
 
 
-class TestPurgeNoCache(TestPurgeBase):
-    def setUp(self):
-        self.set_cache_val(0)
+class TestPurgeInvalidConfig(TestPurgeBase):
```

Comment:
```suggestion
class TestPurgeInvalidConfig(tester.TempestaTest):
```
You should use `TempestaTest`. You don't use anything from the parent class.